### PR TITLE
CICS language for COBOL - syntax analysis

### PR DESCRIPTION
**Pull request by sergiuilie**

### CODE REVIEW

**Comment by sergiuilie** [comment]
@grianbrcom @asatklichov

**Comment by temanbrcom** [comment]
@grianbrcom 
@asatklichov

File: server/src/test/java/com/ca/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
   private static final String MESSAGE =
       "Extraneous input '<EOF>' expected {ACCEPT, ADD, ALTER, CALL, CANCEL, CLOSE, "
           + "COMPUTE, CONTINUE, DELETE, DISABLE, DISPLAY, DIVIDE, ENABLE, ENTRY, "
-          + "EVALUATE, EXEC, EXHIBIT, EXIT, GENERATE, GOBACK, GO, IF, INITIALIZE, "
+          + "EVALUATE, EXEC, EXHIBIT, EXIT, GENERATE, GO, GOBACK, IF, INITIALIZE, "
```
Comment (asatklichov) [inline server/src/test/java/com/ca/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java:40]: Is it necessary to provide sorted expected values ?

File: server/src/test/java/com/ca/lsp/cobol/usecases/TestPerformWithoutEndNotCauseNPE.java [hunk 2]
```diff
@@ -49,10 +49,10 @@ public void test() {
   @Override
   protected void assertDiagnostics(List<Diagnostic> diagnostics) {
     {
-      assertEquals(1, diagnostics.size(), "Number of diagnostics");
+      assertEquals(2, diagnostics.size(), "Number of diagnostics");
```
Comment (asatklichov) [inline server/src/test/java/com/ca/lsp/cobol/usecases/TestPerformWithoutEndNotCauseNPE.java:52]: Optional (all tests):  Hamcrest way - assertThat("Number of diagnostics", diagnostics, hasSize(2));

**Review by asatklichov — APPROVED** [review]
🐬

File: server/src/test/java/com/ca/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java (hunk 1 above)
Comment (temanbrcom) [reply server/src/test/java/com/ca/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java:40]: They are not really sorted, it is the order of the rules in the lexer grammar. And the best practice to keep them sorted when it is possible

File: server/src/test/java/com/ca/lsp/cobol/usecases/TestPerformWithoutEndNotCauseNPE.java (hunk 2 above)
Comment (temanbrcom) [reply server/src/test/java/com/ca/lsp/cobol/usecases/TestPerformWithoutEndNotCauseNPE.java:52]: I just found an issue with determining the error positions, so I'll fix it as a separate activity and rewrite the test

**Review by zacanbrcom — APPROVED** [review]
🥳

**Pull request merged**